lib/reverse_sentence.rb

```diff
@@ -1,6 +1,61 @@
 # A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n ^2) because lines 47-55 contain a nested loop.
+# Space complexity: ? O(n) because of the empty_index_positions array from lines 33 - 38
```

Comment:
I would say O(m) where m is the number of words, but yet.  Could you instead do this with O(1) space complexity?